contribution/polars_ops_multiple_arguments/src/lib.rs

```diff
@@ -0,0 +1,72 @@
+use std::ops::Add;
+
+use polars::export::arrow::array::*;
+use polars::export::arrow::compute::arity::binary;
+use polars::export::arrow::types::NativeType;
+use polars::prelude::*;
+use polars_core::utils::{align_chunks_binary, combine_validities_or};
+use polars_core::with_match_physical_numeric_polars_type;
+
+// Prefer to do the compute closest to the arrow arrays.
+// this will tend to be faster as iterators can work directly on slices and don't have
+// to go through boxed traits
+fn compute_kernel<T>(arr_1: &PrimitiveArray<T>, arr_2: &PrimitiveArray<T>) -> PrimitiveArray<T>
+where
+    T: Add<Output = T> + NativeType,
+{
+    // process the null data separately
+    // this saves an expensive branch and bitoperation when iterating
+    let validity_1 = arr_1.validity();
+    let validity_2 = arr_2.validity();
+
+    let validity = combine_validities_or(validity_1, validity_2);
+
+    // process the numerical data as if there were no validities
+    let values_1: &[T] = arr_1.values().as_slice();
+    let values_2: &[T] = arr_1.values().as_slice();
```

Comment:
I think `arr_1` may be a typo and should be `arr_2` here
```python
let values_2: &[T] = arr_2.values().as_slice();
```

Reply:
Yeap, can you make a PR?